media/css/mozorg/diversity/2021/diversity-2021.scss

```diff
@@ -0,0 +1,475 @@
+    right: 3.75rem;
+
+    .hide-next & {
+        display: none;
+    }
+
+    .c-modal-button-next {
+        background-image: url('/media/protocol/img/icons/arrow-right.svg');
+    }
+}
+
+.c-modal-prev {
+    @extend .mzp-c-modal-close; /* stylelint-disable-line scss/at-extend-no-missing-placeholder  */
+    right: 7.1rem;
+
+    .hide-next & {
+        right: 3.75rem;
+    }
+
+    .hide-prev & {
+        display: none;
+    }
+
+    .c-modal-button-prev {
+        background-image: url('/media/protocol/img/icons/arrow-left.svg');
+    }
+}
+
+.c-modal-button-next,
+.c-modal-button-prev {
+    @extend .mzp-c-modal-button-close; /* stylelint-disable-line scss/at-extend-no-missing-placeholder */
+}
+
+.c-section-intro-bg {
+    @media #{$mq-lg} {
+        background: url('/media/img/foundation/annualreport/2019/moz_site_som_fighting_for_people_left.svg') left center no-repeat, url('/media/img/foundation/annualreport/2019/moz_site_som_fighting_for_people_right.svg') right center no-repeat;
+        background-size: auto 100%;
+    }
+}
+
+.mzp-l-content + .c-section-intro-bg {
+    margin-top: $spacing-2xl;
+}
+
+.c-section-intro {
+    margin-top: $layout-xl;
+}
+
+.c-section-wrapper {
+    margin: 0 auto;
+    padding-top: $layout-sm;
+    max-width: $content-lg;
+
+    h2 {
+        @include text-title-lg;
+        font-weight: 100;
+        text-align: center;
+    }
+
+    h3 {
+        @include text-title-sm;
+        font-weight: 600;
+        text-align: left;
+    }
+
+    h4 {
+        @include text-title-xs;
+        font-weight: 600;
+        text-align: left;
+    }
+
+    .c-subtitle {
+        text-align: center;
+    }
+
+    ul {
+        list-style-type: disc;
+
+        li {
+            margin-bottom: $spacing-sm;
+        }
+    }
+}
+
+.bold-item {
+    font-weight: 600;
+
+    ul {
+        font-weight: 400;
+    }
+}
+
+.c-article-list {
+    .mzp-l-card-quarter,
+    .mzp-l-card-half {
+        .mzp-c-card-title {
+            @include text-title-xs;
+        }
+    }
+
+    .mzp-l-card-hero {
+        .mzp-c-card + .mzp-c-card {
+            .mzp-c-card-title {
+                @include text-title-xs;
+            }
+        }
+    }
+
+    .mzp-c-card-feature-title,
+    .mzp-c-card-title {
+        font-weight: 100;
+    }
+
+    .mzp-c-card-feature .mzp-c-card-feature-title {
+        @include text-title-md;
+
+        @media #{$mq-lg} {
+            @include text-title-lg;
+        }
+    }
+
+    .mzp-c-card-feature {
+        margin-bottom: $spacing-lg;
+    }
+
+    .mzp-c-card {
+        background-color: transparent;
+
+        &:hover {
+            background-color: $color-white;
+        }
+    }
+}
+
+.c-section-graphs {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+
+    &.graph-center {
+        justify-content: center;
+    }
+
+    figure {
+        border: 1px solid $color-marketing-gray-30;
+        margin: $spacing-md 0;
```

Comment:
I think these might look better if they're centered in small viewports: `margin: $spacing-md auto;`